### ISSUE TITLE
Performance of LSTR using camera-only modality

### ISSUE THREAD

**Issue by priyamdey** [amazon-research/long-short-term-transformer#8]
Hello. Can you share the camera-only performance of LSTR on the TVSeries and THUMOS dataset? 

Thanks

**Comment by xumingze0308** [comment]
Using our default settings described in the paper, we got 56.8 mAP on THUMOS and 83.3% mcAP on TVSeries.

**Comment by Echo0125** [comment]
> Hello. Can you share the camera-only performance of LSTR on the TVSeries and THUMOS dataset?
> 
> Thanks

Excuse me, could you please tell me  your performance on TVSeries? I only get 57.96% mcAP on TVSeries while ignoring the background.

**Comment by priyamdey** [comment]
@Wangjh0125 , I got around 82.2% (background excluded) on TVSeries using the pretrained RN50 backbone referred in the README and default settings of the LSTR codebase (I couldn't find any differences in the defaults of the paper and  the defaults of the code).

**Comment by Echo0125** [comment]
> @Wangjh0125 , I got around 82.2% (background excluded) on TVSeries using the pretrained RN50 backbone referred in the README and default settings of the LSTR codebase (I couldn't find any differences in the defaults of the paper and the defaults of the code).

Thanks, I don't know why I get the mcAP so low; maybe my operation has some mistakes...

**Comment by priyamdey** [comment]
I was too facing similar issues with low mcAP (although around ~70). You can make sure a few things are right:

- frames extracted at `24fps` from each video (I used `ffmpeg`)

- Features extracted properly using the pretrained RN50 backbone ([ckpt](https://download.openmmlab.com/mmaction/recognition/tsn/tsn_r50_320p_1x1x8_100e_kinetics400_rgb/tsn_r50_320p_1x1x8_100e_kinetics400_rgb_20200702-ef80e3d7.pth), [code](https://github.com/open-mmlab/mmaction2/blob/master/configs/recognition/tsn/tsn_r50_320p_1x1x8_100e_kinetics400_rgb.py)). Main thing to focus here is the transformation: Resize frames to 320 (shorter side, this is how the PT backbone was trained) and normalize simply using IN1K mean/std. Transform would look like this: `Compose(Resize(320), Normalize(mean=(0.485, 0.456, 0.406), std=(0.228, 0.224, 0.225))`. Avgpool, flatten and dump the 1D features. I had put model in eval mode while doing all these.

See if these help you in getting to that desired number.

**Comment by Echo0125** [comment]
It's very kind of you! I'll try it at a later time. Thanks again!

**Comment by Wangjh0125** [comment]
I try it again, but sadly get 55.7% mcAP, and the training loss is not converged...Maybe my target_perframe.py has some mistakes. Could you please tell me how you got it? Thanks
